### PR TITLE
Liya w2 JavaScript  

### PR DESCRIPTION
**Pull request by liya-oz**

### CODE REVIEW

File: 1-JavaScript/Week2/assignment/ex7-mindPrivacy.js [hunk 1]
```diff
@@ -29,18 +29,29 @@ const employeeRecords = [
...
+    filteredEmployees.push({ name, occupation, email });
+  }
+  return filteredEmployees;
 }
 
 // ! Test functions (plain vanilla JavaScript)
 function test1() {
-  console.log('Test 1: filterPrivateData should take one parameters');
+  console.log(filterPrivateData(employeeRecords));
   console.assert(filterPrivateData.length === 1);
```
Comment (reposman33) [inline 1-JavaScript/Week2/assignment/ex7-mindPrivacy.js:45]: I would say the filtered data consists of 2 employee records with only non-private data (`name`, `occupation` and `email`). Not sure if this is an error in te assignment?

**Review by reposman33 — COMMENTED** [review]
Not all assignments are finished - see nr 6, `totalCost`

**Review by reposman33 — COMMENTED** [review]
I miss the tests for assignments 4,5 and 6.

File: 1-JavaScript/Week2/assignment/ex7-mindPrivacy.js [hunk 2]
```diff
@@ -29,18 +29,29 @@ const employeeRecords = [
...
 }
 
 function test2() {
-  console.log('Test 2: gender and salary should be filtered out');
+  employeeRecords.forEach((employee) => {
```
Comment (reposman33) [inline 1-JavaScript/Week2/assignment/ex7-mindPrivacy.js:49]: Is logging the contents of the employees part of the test?

File: 1-JavaScript/Week2/assignment/ex7-mindPrivacy.js [hunk 3]
```diff
@@ -29,18 +29,29 @@ const employeeRecords = [
...
+    const { name, occupation, email } = employees[i];
+    filteredEmployees.push({ name, occupation, email });
+  }
+  return filteredEmployees;
 }
 
 // ! Test functions (plain vanilla JavaScript)
 function test1() {
-  console.log('Test 1: filterPrivateData should take one parameters');
+  console.log(filterPrivateData(employeeRecords));
```
Comment (reposman33) [inline 1-JavaScript/Week2/assignment/ex7-mindPrivacy.js:44]: Is logging the array of filtered employee records part of the test?

**Review by reposman33 — APPROVED** [review]

**Comment by liya-oz** [comment]
**Thank you for your reviewing, @reposman33 !**

Here is how I implemented your corrections:

> Not all assignments are finished - see nr 6, `totalCost`

Ex 6, `totalCost`is now complete. 

>small typo: arrow (->) should be array.

The typo is corrected too.

> Is logging the array of filtered employee records part of the test?

Console.logging is part of the tests but I commented out that section of the code and now console output looks much clearer. 

> It works fine . A bit complicated way to say 'if there are more than 3 items delete the first item in the array'.

I simplified it by using one of the array methods.

**Pull request closed without merging**